src/python/grpcio_tests/tests_aio/unit/sync_server.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import multiprocessing
+
+from concurrent import futures
+from time import sleep
+
+import grpc
+from src.proto.grpc.testing import messages_pb2
+from src.proto.grpc.testing import test_pb2_grpc
+
+# TODO (https://github.com/grpc/grpc/issues/19762)
+# Change for an asynchronous server version once it's implemented.
+
+
+class TestServiceServicer(test_pb2_grpc.TestServiceServicer):
+
+    def UnaryCall(self, request, context):
+        return messages_pb2.SimpleResponse()
+
+
+class Server(multiprocessing.Process):
+    """
+    Synchronous server is executed in another process which initializes
+    implicitly the grpc using the synchronous configuration. Both worlds
+    can not coexist within the same process.
+    """
+
+    PORT = 3333
```

Comment:
this would mean that this port would need to be sent back to the test since it is used for connecting to the server. Despite the changes required for coordinating the fixture and the test would not be rocket science, IMO they are still none negligible considering that this fixture will be removed once we have the asynchronous version of the server.

Reply:
Using a fixed port will introduce a flake in our tests, internally, they are ran in parallel. The port number can be conflicted. If you want to pick a random port, you can try this piece of code I wrote for testing. Or use the `portpicker` package.

```Python
@contextmanager
def get_free_loopback_tcp_port():
    if socket.has_ipv6:
        tcp_socket = socket.socket(socket.AF_INET6)
    else:
        tcp_socket = socket.socket(socket.AF_INET)
    tcp_socket.bind(('', 0))
    address_tuple = tcp_socket.getsockname()
    yield "localhost:%s" % (address_tuple[1])
    tcp_socket.close()
```

Reply:
Thanks, I will use this.